api/src/main/java/org/apache/cloudstack/api/command/admin/offering/UpdateDiskOfferingCmd.java

```diff
@@ -77,6 +77,13 @@
             since = "4.13")
     private String zoneIds;
 
+    @Parameter(name = ApiConstants.TAGS,
+            type = CommandType.STRING,
+            description = "comma-separated list of tags for the disk offering, tags should match with existing storage pool tags",
+            since = "4.15")
+    private String tags;
```

Comment:
authorized = {RoleType.Admin}

?

Reply:
code has been updated, thanks for the hint @weizhouapache